### DISCUSSION TITLE
OpenAPI and AsyncAPI Integration

### DISCUSSION THREAD

**Discussion by TorbenHammes** [outline/outline#5218] [Ideas]
Hey there, I've seen that there is already a feature request for OpenAPI Integration.
Sometimes I work with AsyncAPI https://www.asyncapi.com/ and that could be interesting, too.

It there a plan for an OpenAPI integration and could it be extended with AsyncAPI?

Greetings from Berlin
Torben ✌️

**Comment by apoorv-mishra** [comment]
Related to https://github.com/outline/outline/discussions/4972
